### PR TITLE
Features/profile page

### PR DESCRIPTION
**Pull request by yuryvakolov**
Practice PR
* Angular Best Practices App
*  Components-library
* Storybok implementation

### CODE REVIEW

File: projects/lib-badges/src/lib/badges/badges.component.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="wrapper">
```
Comment (yuryvakolov) [inline projects/lib-badges/src/lib/badges/badges.component.html]: remove div with class="wrapper"

File: projects/lib-badges/src/lib/badges/badges.component.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
+@import "src/styles/theme";
+.wrapper {
```
Comment (yuryvakolov) [inline projects/lib-badges/src/lib/badges/badges.component.scss]: change .wrapper on :host

File: projects/lib-badges/src/lib/badges/badges.component.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { Component, Input } from '@angular/core';
+import { Badge } from '../services/badges.service';
```
Comment (yuryvakolov) [inline projects/lib-badges/src/lib/badges/badges.component.ts]: fix path from src

File: projects/lib-badges/src/lib/lib-badges.component.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+@Component({
+  selector: 'lib-badges-lib-badges',
+  template: `
+    <lib-badges-badges *ngFor="let badge of badges" [badge]="badge"></lib-badges-badges>
+  `,
+  styles: []
+})
+export class LibBadgesComponent implements OnInit {
+  public badges = this.badgesService.getBadges();
```
Comment (yuryvakolov) [inline projects/lib-badges/src/lib/lib-badges.component.ts]: create getter

File: src/layouts/user/pages/profile-page/components/card-skills/card-skills.component.scss [hunk 5]
```diff
@@ -1,7 +1,7 @@
 @import 'src/styles/theme';
 
 .wrapper {
-  width: 350px;
+  width: 390px;
```
Comment (yuryvakolov) [inline src/layouts/user/pages/profile-page/components/card-skills/card-skills.component.scss]: change px => em. (px2em)

File: src/layouts/user/pages/profile-page/profile-page.component.html [hunk 6]
```diff
@@ -32,6 +32,7 @@ <h3>Skills Acquired (18)</h3>
       </div>
       <div class="right-block">
         <span>Badges (10)</span>
+<!--        <lib-badges-lib-badges></lib-badges-lib-badges>-->
```
Comment (yuryvakolov) [inline src/layouts/user/pages/profile-page/profile-page.component.html]: change name

File: src/layouts/user/pages/profile-page/profile-page.component.html [hunk 7]
```diff
@@ -32,6 +32,7 @@ <h3>Skills Acquired (18)</h3>
       </div>
       <div class="right-block">
         <span>Badges (10)</span>
```
Comment (yuryvakolov) [inline src/layouts/user/pages/profile-page/profile-page.component.html]: add badjes.lengh

Comment (jrluka13) [reply src/layouts/user/pages/profile-page/profile-page.component.html]: aproove

File: src/layouts/user/pages/profile-page/components/card-skills/card-skills.component.scss (hunk 5 above)
Comment (jrluka13) [reply src/layouts/user/pages/profile-page/components/card-skills/card-skills.component.scss]: aproove

File: projects/lib-badges/src/lib/lib-badges.component.ts (hunk 4 above)
Comment (jrluka13) [reply projects/lib-badges/src/lib/lib-badges.component.ts]: aproove

File: projects/lib-badges/src/lib/badges/badges.component.ts (hunk 3 above)
Comment (jrluka13) [reply projects/lib-badges/src/lib/badges/badges.component.ts]: aproove

File: projects/lib-badges/src/lib/badges/badges.component.scss (hunk 2 above)
Comment (jrluka13) [reply projects/lib-badges/src/lib/badges/badges.component.scss]: aproove

File: projects/lib-badges/src/lib/badges/badges.component.html (hunk 1 above)
Comment (jrluka13) [reply projects/lib-badges/src/lib/badges/badges.component.html]: aproove

File: src/layouts/user/pages/profile-page/profile-page.component.html (hunk 6 above)
Comment (jrluka13) [reply src/layouts/user/pages/profile-page/profile-page.component.html]: approve

**Comment by yuryvakolov** [comment]
rebased and merge

**Pull request merged**